File: java/src/main/java/org/example/queue/QueueRun.java
package org.example.queue;

public class QueueRun {
    public static void run() {
        Queue queue = new Queue();
        queue.push("qwd");
        queue.push("llwq");
        queue.push("212");
        queue.push("fgg");

        System.out.println(queue.front());
        queue.pop();

        System.out.println(queue.front());
        queue.pop();

        System.out.println(queue.front());
    }
}

File: php/src/Queue/QueueRun.php
<?php

namespace Claudiosilvajunior\Php\Queue;

class QueueRun
{
    public static function run(): void
    {
        $queue = new Queue();
        $queue->push(1);
        $queue->push(21);
        $queue->push('2131');

        var_dump($queue->front());
        $queue->pop();

        var_dump($queue->front());
        $queue->pop();

        var_dump($queue->front());
    }
}